.gitignore

```diff
@@ -4,3 +4,4 @@
 
 build
 source/locale/pot/.doctrees
+.vscode/settings.json
```

Comment:
This doesn't belong into this project's gitignore because the file is unrelated to this repository. You can put OS/IDE files into a global git ignore file instead: https://medium.com/self-modifying-code/create-a-global-gitignore-step-by-step-for-macos-and-windows-31a765291409

Reply:
Yeah, sorry I wasn't clear about the _global_ gitignore

Reply:
Alright. Let me correct this.